src/wire/udp.rs

```diff
@@ -28,6 +28,8 @@ mod field {
     }
 }
 
+pub const HEADER_LEN: usize = field::CHECKSUM.end;
```

Comment:
Should we use this constant below instead of `field::CHECKSUM.end` for clarity?

Reply:
Done